### ISSUE TITLE
ERROR: Cannot install -r requirements.txt (line 2) and pathway because these package versions have conflicting dependencies.

### ISSUE THREAD

**Issue by arjun-ms** [pathway-labs/dropbox-ai-chat#3]
![image](https://github.com/pathway-labs/dropbox-ai-chat/assets/64315213/5594af06-2b63-4d87-a504-78d2a3dbba5b)

**Comment by chriz-ty** [comment]
I'm also experiencing the same issue when I was trying to install the requirements.

**Comment by munish0838** [comment]
Facing the same issue

**Comment by zxqfd555-pw** [comment]
Hello @arjun-ms @chriz-ty @munish0838.

Could you please let us know what Python version you're using? You can check it by executing `python --version` in a command line.

I have checked the dependencies of the llm_app library and I see that they refer to certain versions of pathway. It may be the case that the version you have installed is too old. Could you try to uninstall `pathway` and `llm_app` and run the `pip install -r requirements.txt` again? 

On the other note, using a clean `virtualenv` to install the dependencies should also help.

**Comment by arjun-ms** [comment]
Hey @zxqfd555-pw !

I am using Python 3.10. I believe the error I encountered is due to having installed this on a Windows machine, whereas the pathway package currently supports only Linux and macOS. 

Now, I am attempting to run it on Docker.

**Comment by chriz-ty** [comment]
Yeah, I tried to attempt in docker and it worked! But now there is another problem that I'm facing. The external url and the network url are not working..

**Comment by munish0838** [comment]
Are you using wsl? @chriz-ty

**Comment by chriz-ty** [comment]
> Are you using wsl? @chriz-ty

Nope

**Comment by abhibarnwal707** [comment]
> Hello @arjun-ms @chriz-ty @munish0838.
> 
> Could you please let us know what Python version you're using? You can check it by executing `python --version` in a command line.
> 
> I have checked the dependencies of the `llm_app` library and I see that they refer to certain versions of `pathway`. It may be the case that the version you have installed is too old. Could you try to uninstall `pathway` and `llm_app` and run the `pip install -r requirements.txt` again?
> 
> On the other note, using a clean `virtualenv` to install the dependencies should also help.

Hii @zxqfd555-pw , my python version is Python 3.12.1
Still finding the same error, even tried to make a new environment for it

**Comment by stefan52a** [comment]
see conclusion of https://github.com/pathway-labs/dropbox-ai-chat/issues/5
